src/components/Heading/Heading.module.css

```diff
@@ -81,15 +81,13 @@
   color: var(--eds-theme-color-text-brand-primary);
 }
 .heading--info {
-  /* TODO(brand-tokens): remove */
```

Comment:
we're actually going to keep the `info-` tokens around for now until we figure out a deprecation plan. this style will be removed once the "info" variant is removed for Heading & Text

Reply:
Maybe update the text in parens then?

Reply:
I thought about that, but I found it confusing since you still need to reference the JS to double-check if the variant was removed. But when we do remove `info` from the JS it's not likely we'd forget to remove the styling